queue_worker.rb

```diff
@@ -82,6 +80,10 @@ def process(task:)
 
           work_block.call
 
+          # If execution reaches this point, we can ensure that no exceptions were raised.
```

Comment:
Even though this is totally true, let's keep this sort of thing in 1 place, in the `finish_task` method. That way we don't change the task object state in multiple places.

Reply:
The issue is that if I set the `finished_successfully` there, it'd be always true, because `finish_task` is always called

Reply:
Ah, right. Good point 👍